src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -90,14 +90,17 @@ run scripts or interact programmatically against Dockstore APIs, and [run workfl
     `;
 
     this.textDataUbuntuLinux = `
-#### Part 1 - Install dependencies
-1. Install Java 17 (This example installs OpenJDK 17)
+#### Part 1 - Install Java dependencies
```

Comment:
Could probably just be `Install Java` at this point since there's a header added before Docker